R/translate_package.R

```diff
@@ -73,7 +73,6 @@ translate_package = function(
     if (tolower(prompt('Exit now to repair any of these? [y/N]')) %chin% c('y', 'yes')) return(invisible())
   }
 
-  if (verbose) message('Generating .pot files...')
```

Comment:
is this removed intentionally? not quite seeing the relation to the rest of the PR

Reply:
Oh NVM, I see it's just subsumed in `write_po_files()`